connector/docs/config.md

```diff
@@ -1,33 +1,35 @@
 # Configuring Salesforce Operations
 
-[[Importing the Salesforce Certificate]](#importing-the-salesforce-certificate)  [[Common parameters]](#common-parameters) [[Salesforce Operations]](#salesforce-perations) [[Logging out of Salesforce]](#logging-out-of-salesforce)
+[[Prerequisites]](#Prerequisites) [[Salesforce Operations]](#salesforce-operations) [[Logging out of Salesforce]](#logging-out-of-salesforce)
 
-## Importing the Salesforce Certificate
+## Prerequisites
+
+### Importing the Salesforce Certificate
 
 To use the Salesforce connector, add the <salesforce.init>  element to your configuration before carrying out any other Salesforce operations.
 
-Before you start configuring the connector, import the Salesforce certificate to your ESB client keystore.
+Before you start configuring the connector, import the Salesforce certificate to your EI client keystore.
```

Comment:
Before you start configuring the connector, import the Salesforce certificate to your WSO2 Enterprise Integrator (EI) client keystore.

P.S: We need to introduce the abbreviation at the beginning of each page so users know what we are referring to :)